app/views/user/index.html.erb

```diff
@@ -1,17 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blog App</title>
+    <link rel="stylesheet" href="application.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">   
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=5.7.0/css/font-awesome.min.css">     
+     
+    
+</head>
+
 <h1> Posts for given user </h1>
 
 <h1>Blog App</h1>
 
- <% if @posts.blank? %>
+ <% if @users.blank? %>
  <p>There are not any posts currently in the system.</p>
  <% else %>
- <p>These are the current posts in our system</p>
-<% @posts.each do |c| %>
-<ul id = "posts">
+       <% @users.each do |d| %>
+ 
+   <div class = "user one">
+    <img class= 'user img' src='<%= d.photo %>'>
```

Comment:
- [ ] Kindly make sure in all wireframes where the user photo is supposed to appear, it is shown correctly like on their expected wireframes. The user is coming from your styling where you give the `user.img` a `width: 0%`. 
Make sure you update in all wireframes and style it corrctly